LineageABCmodel.jl

```diff
@@ -3304,7 +3314,7 @@ function analysemultipleABCstatistics( lineagetree::Lineagetree, state_chains_hi
     end     # end if model with hiddenmatrix
     # ...for standardised event-times histogram:
     if( withgraphical )
-        if( (mymodel==1) || (mymodel==2) || (mymodel==3) || (mymodel==4) || (mymodel==9) || (mymodel==11) || (mymodel==12) || (mymodel==13) || (mymodel==14) )  # only meaningful, if a model with rescaled time
+        if mymodel in keys(ALLOWED_MODELS) # only meaningful, if a model with rescaled time

```

Comment:
@pembacher I already did this kind of changes, multiple times, in #3 (just [one example](https://github.com/Jamie-Dean-Lab/Lineage-analysis/pull/3/files#diff-e3e06caf0eeb240de3dce2a5a130ced006c54deb2e026d8accdfd397ff1ec69fR3027)), I had to repeat the same changes again, in multiple places, because my changes were overwritten by your later commits.  In a collaborative setting this slows down development sensibly.